curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/64649b108df035051cb2ba2d.md

```diff
@@ -13,16 +13,14 @@ And with those changes, your freeCodeCamp forum leaderboard project is now compl
 
 # --hints--
 
-Your opening anchor tag should have a `target` attribute set to `_blank`.
 
-```js
-assert.match(code, /<a\s+(?:class=['"]post-title['"]\s+target=['"]_blank['"]|target=['"]_blank['"]\s+class=['"]post-title['"])(?:.|\n)*?<\/a>/);
-```
-
-Your opening anchor tag should have a `href` attribute set to `${forumTopicUrl}${slug}/${id}` after the `target` attribute.
+Verify the code includes a properly configured href attribute (${forumTopicUrl}${slug}/${id}) and a target attribute (_blank) within the opening <a> tag.
```

Comment:
This change is missing backticks around certain keywords and should follow the same language convention setup for writing hint text.

let's go with this change

```suggestion
Your opening anchor tag should have an `href` attribute set to `${forumTopicUrl}${slug}/${id}`.
```

Reply:
Got it @jdwilkin4, working on it :raised_hands: